### DISCUSSION TITLE
Help x3

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#181] [Q&A] [answered]
Hello, and sorry, how to delete a item in item.yml? is not possible give me a error...

**Comment by GoodestEnglish** [comment, marked as answer]
You cannot delete an item from item.yml. However you can set the item slot to -1 so it will not give the item.

**Reply by liexes** [reply]
And to add one?

**Reply by GoodestEnglish** [reply]
Currently you can't. Might add this feature soon
